test/hotspot/jtreg/compiler/lib/generators/UniformDoubleGenerator.java

```diff
@@ -35,6 +35,7 @@ final class UniformDoubleGenerator extends UniformIntersectionRestrictableGenera
      */
     public UniformDoubleGenerator(Generators g, double lo, double hi) {
         super(g, lo, hi);
+        if (Double.compare(lo, hi) >= 0) throw new EmptyGeneratorException();
```

Comment:
I suggest to add braces

```suggestion
        if (Double.compare(lo, hi) >= 0) {
            throw new EmptyGeneratorException();
        }
```